Update imports for lilac

https://github.com/eol-uchile/edx_xblock_scorm/blob/eb70cc20f18f60439c4970110bb37dbad7fe1b71/scormxblock/tests.py#L4

del test automatizado https://github.com/eol-uchile/edx_xblock_scorm/actions/runs/19246096348/job/55020396264#step:6:518

```
==================================== ERRORS ====================================
____________________ ERROR collecting scormxblock/tests.py _____________________
scormxblock/tests.py:4: in <module>
    from util.testing import UrlResetMixin
../../edx-platform/import_shims/lms/util/__init__.py:6: in <module>
    warn_deprecated_import('util', 'common.djangoapps.util')
../../edx-platform/import_shims/warn.py:37: in warn_deprecated_import
    raise DeprecatedEdxPlatformImportError(old_import, new_import)
E   import_shims.warn.DeprecatedEdxPlatformImportError: Importing util instead of common.djangoapps.util is deprecated
```

--- a/scormxblock/tests.py
+++ b/scormxblock/tests.py
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 import json
 
-from util.testing import UrlResetMixin
+from common.djangoapps.util.testing import UrlResetMixin
 from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
 from opaque_keys.edx.keys import CourseKey
 from ddt import ddt, data
@@ -9,7 +9,7 @@ from django.test import Client
 from freezegun import freeze_time
 from lms.djangoapps.instructor_task.models import InstructorTask
 from .task import scorm_task
-from student.tests.factories import UserFactory
+from common.djangoapps.student.tests.factories import UserFactory
 import mock
 from xblock.field_data import DictFieldData
 

--- a/setup.py
+++ b/setup.py
@@ -22,7 +22,7 @@ def package_data(pkg, roots):
 
 setup(
     name='scormxblock-xblock',
-    version='0.5',
+    version='0.5.1',
     description='scormxblock XBlock',   # TODO: write a better description.
     packages=[
         'scormxblock',

